src/App.js

```diff
@@ -127,6 +129,10 @@ export function App() {
               element={<TypologyDropdownDemo />}
             />
           </Route>
+          <Route
```

Comment:
Can we remove this example and move it to a storybook ?

Reply:
Sure thing, will refer to the existing Storybook files we have